modules/idaithalam/src/main/java/io/virtualan/idaithalam/contract/IdaithalamExecutor.java

```diff
@@ -109,16 +101,23 @@ public static int validateContract(String featureHeading)
      * @throws UnableToProcessException the unable to process exception
      */
     public static int validateContract(String featureHeading, ApiExecutorParam apiExecutorParam)
-        throws UnableToProcessException {
+            throws UnableToProcessException {
         byte exitStatus;
         try {
-            String fileIndex = UUID.randomUUID().toString();
             VirtualanClassLoader classLoaderParnet = new VirtualanClassLoader(IdaithalamExecutor.class.getClassLoader());
             ExecutionClassloader classLoader = addConfToClasspath(classLoaderParnet, apiExecutorParam.getOutputDir());
-            generateFeatureFile(classLoader, apiExecutorParam);
+            /** @author Oliver Glas. */
+            if (apiExecutorParam.getExecution() == null || apiExecutorParam.getExecution().equalsIgnoreCase("execute") == false) {
```

Comment:
Change the condition with 
if(apiExecutorParam.getExecution() == null || apiExecutorParam.getExecution().equalsIgnoreCase("both") ||  apiExecutorParam.getExecution().equalsIgnoreCase("generate") ) {

Reply:
which implies that will not work for execute choice

Reply:
I checked for execute on purpose.
The disadvantage if you don't do that, is that it would apply for everything what is not "both" or "generate".
Including "generatex" or "bots" or any other typo.

Reply:
I would not check for "both" because whatever you do not check explicitely, will be default. Default should be that both steps are done. Like if you put "all" instead of "both" will work this way, but not if you check for "both".